util.py

```diff
@@ -127,3 +136,14 @@ def __remote_has_newer(installed: str, remote: Union[str, None]) -> bool:
 
 def helper_function(function_name: str) -> bool:
     return base_logic.call_function_by_name(function_name)
+
+
+def check_if_version_exists(installed: str, remote: str, is_remote: bool) -> str:
+    if is_remote:
+        if remote == "0.0.0":
+            return "unknown"
+        return remote
+    else:
+        if installed == "0.0.0":
+            return "unknown"
+        return installed
```

Comment:
Probably a misunderstanding, waht i meant was something like this:
```suggestion
    if "0.0.0" in [remote,installed]:
        return "unknown"
    if is_remote:
        return remote
    else:
        return installed
```


so if either of them is 0.0.0 it shows "unknown"